### DISCUSSION TITLE
use on Raspberry PI

### DISCUSSION THREAD

**Discussion by GermanJack** [tryphotino/photino.NET#176] [Q&A] [answered]
Hello,
I need a version of TryPhotino.net which is running on RaspBerry OS  64 Bit.
The delivered runtime for ARM64 is not running on the Paspi.
It brings a error that a TryPhotino.native cannot find a required dependency.
Even after installing GTK3.
What kind of dependency is missing and how to get it working RaspBerry OS 64 Bit?

**Comment by MikeYeager** [comment, marked as answer]
@GermanJack Please follow the publishing guide. You can find it in the Photino.Samples repo in the debug branch. Refer to the Debian example.
